controllers/CommentController.js

```diff
@@ -26,5 +26,17 @@ res.status(200).json({getComments , totalPages:Math.ceil(count / limit), current
 
 }
 
+const deleteComment = async (req , res) =>{
+  const newDeleteComment = await Comment.findById(req.params.id)
+  if(newDeleteComment.postBy._id.toString() === req.user._id.toString()){
+    await Comment.findByIdAndDelete(req.params.id)
+    res.status(200).json("Comment Deleted")
```

Comment:
Remember when using `res.json`, send serializable objects only

And you can just send a 204 (No content) status code alone